makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
```

Comment:
For now umbrella should also require that the binary is specified. I understand that there may be cases when umbrella exists locally, but in Makeflow we assume that you are specifying the executable files in most cases. This also ensures consistent version across nodes.